connman/src/connection.c

```diff
@@ -1032,14 +1044,15 @@ bool __connman_connection_update_gateway(void)
 		if (active_gateway->ipv4_gateway &&
 				active_gateway->ipv4_gateway->active) {
 
+			DBG("unset ipv4");
 			unset_default_gateway(active_gateway,
 						CONNMAN_IPCONFIG_TYPE_IPV4);
 			updated = true;
 		}
 
 		if (active_gateway->ipv6_gateway &&
 				active_gateway->ipv6_gateway->active) {
-
+			DBG("unset ipv6");
```

Comment:
Nitpick: in this case line is replaced but above new line is added in a similar case, not necessary to fix, just a slight inconsistency.

Reply:
Removed these as they were not really necessary DBGs.